### ISSUE TITLE
Threshold for marginal power flows

### ISSUE THREAD

**Issue by ThomDietrich** [reptilex/tesla-style-solar-power-card#115]
Hey there, occasionally I can see that a very small power is registered for one of the entities. I've observed this for generation, battery, grid, and the electric car charger.

Example:
![image](https://user-images.githubusercontent.com/2870104/162427705-c8c6b0d1-0289-4afc-9e07-2291e180a3bc.png)

Would it be possible to define a configuration varible for this? All values small than that value will be replaced by 0.

```yaml
threshold_minimum_power_w: 5
```

Thanks for considering! Cheers

**Comment by reptilex** [comment]
yeah, Ive been thinking about that one too ... will take time unless you do your own PR ;)

**Comment by ThomDietrich** [comment]
Thanks!
I'm not strong in js, let alone ts - I'll leave this in your capable hands :D
